### PR TITLE
Calculate Listing Cost

### PR DESCRIPTION
**Pull request by kevinrodriguez-io**
On the last screen of listing in the area below copies put together a price summary of creating the listing that should: 
Show total cost of solana storage by getting the rent exempt fee for all accounts being created by the listing.
Show transaction cost with # transactions * per transaction cost.

### CODE REVIEW

File: js/packages/web/src/views/auctionCreate/reviewStep.tsx [hunk 1]
```diff
@@ -1,18 +1,82 @@
...
+  const maxExternalAccountSize = connection.getMinimumBalanceForRentExemption(
+    MAX_EXTERNAL_ACCOUNT_SIZE,
+  );
+  const auctionCosts = connection.getMinimumBalanceForRentExemption(
+    MAX_AUCTION_DATA_EXTENDED_SIZE,
+  );
+  // SAFETY DEPOSIT BOX COSTS
+  const safetyDepositBoxCosts = connection.getMinimumBalanceForRentExemption(
+    BASE_SAFETY_CONFIG_SIZE,
+  );
```
Comment (kespinola) [inline js/packages/web/src/views/auctionCreate/reviewStep.tsx:43]: Other accounts created that contribute to call:

- auction cache
-

File: js/packages/web/src/views/auctionCreate/reviewStep.tsx [hunk 2]
```diff
@@ -1,18 +1,82 @@
...
 export const ReviewStep = (props: {
   confirm: () => void;
   attributes: AuctionState;
   setAttributes: Function;
   connection: Connection;
 }) => {
+  const [cost, setCost] = useState(0);
   const item = props.attributes.items?.[0];
 
+  useEffect(() => {
```
Comment (kespinola) [inline js/packages/web/src/views/auctionCreate/reviewStep.tsx:63]: Let's get a loading state in for the cost calculating. 

Toggle load off after all the responses are back.

**Comment by kespinola** [comment]
This is looking good though. How far off are we again?

**Comment by kevinrodriguez-io** [comment]
> This is looking good though. How far off are we again?

We're still like 3x below, it calculates around 2$ on Devnet but it is actually like 7$

**Pull request closed without merging**